app/actions.js

```diff
@@ -149,7 +148,7 @@ export const removeCabal = ({ addr }) => dispatch => {
 // remove cabal
 export const confirmRemoveCabal = ({ addr }) => async dispatch => {
   client.removeCabal(addr)
-  dispatch({ type: 'DELETE_CABAL', addr })
+  // dispatch({ type: 'DELETE_CABAL', addr })
```

Comment:
if it's not needed i guess we should delete the comment?
```suggestion
```

Reply:
This is part of another issue (https://github.com/cabal-club/cabal-desktop/issues/300) where the app would crash upon trying to remove a cabal. Without this line, it prevents the crash and removes it. However the UI does not update. Looks like @wittjosiah has a fix though.

Reply:
alright!